Oqtane.Server/Extensions/PermissionExtension.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using Oqtane.Models;
+
+namespace Oqtane.Extensions
+{
+    public static class PermissionExtension
+    {
+        public static string EncodePermissions(this IEnumerable<Permission> permissionList)
+        {
```

Comment:
Copy from _PermissionRepository Function is static utility and for call is not necessary repository instance.
Refactored as extension.

Reply:
Refactoring of all calls will be done in next step.